packages/zowe-explorer/src/extension.ts

```diff
@@ -517,9 +517,10 @@ function initUSSProvider(context: vscode.ExtensionContext, ussFileProvider: IZow
         )
     );
     context.subscriptions.push(
-        vscode.commands.registerCommand("zowe.uss.deleteNode", async (node: IZoweUSSTreeNode) =>
-            node.deleteUSSNode(ussFileProvider, node.getUSSDocumentFilePath())
-        )
+        vscode.commands.registerCommand("zowe.uss.deleteNode", (node: IZoweUSSTreeNode) => {
+            const tempNode = ussFileProvider.getTreeView().selection[0] as IZoweUSSTreeNode;
+            tempNode.deleteUSSNode(ussFileProvider, tempNode.getUSSDocumentFilePath());
+        })
```

Comment:
Hey @KutluOzel-b, 
You made an observation that this is probably one of the only actions that is still performed from the node instead of calling an API in the ussActions.

Maybe we should consider moving it in a future update, but for now I think it's fine to get the selection from the tree view 😋

Reply:
Thanks for sharing it Fernando :)

Reply:
@KutluOzel-b did we want to create an issue for this or is one already created? this way we don't forget